main.tf

```diff
@@ -18,7 +18,7 @@ resource "aws_launch_configuration" "ecs" {
   instance_type        = "${var.instance_type}"
   key_name             = "${var.key_name}"
   iam_instance_profile = "${aws_iam_instance_profile.ecs_profile.name}"
-  security_groups      = ["${aws_security_group.ecs.id}"]
+  security_groups      = ${concat(["${aws_security_group.ecs.id}"], var.security_group_ids)}
```

Comment:
This is safer: `${compact(concat(["${aws_security_group.ecs.id}"], var.security_group_ids))}`